File: app/src/LightningLite/java/acr/browser/lightning/activity/AdManager.java
package acr.browser.lightning.activity;

import android.app.Activity;
import com.google.android.gms.ads.AdRequest;
import com.google.android.gms.ads.AdSize;
import com.google.android.gms.ads.AdView;
import com.google.android.gms.ads.AdListener;

import android.widget.RelativeLayout;

import acr.browser.lightning.R;
import android.view.ViewGroup;

public class AdManager {
    private Activity mActivity;
    AdView mAdView;

    public AdManager(Activity activity) {
        mActivity = activity;

        ViewGroup contentView = (ViewGroup)activity.findViewById(R.id.ui_layout);
        mAdView = new AdView(activity);
        mAdView.setAdUnitId("ca-app-pub-7879734750226076/6792006747");
        mAdView.setAdSize(AdSize.BANNER);
        mAdView.setAdListener(new AdListener(){});
        RelativeLayout.LayoutParams params = new RelativeLayout.LayoutParams(
                RelativeLayout.LayoutParams.MATCH_PARENT, RelativeLayout.LayoutParams.WRAP_CONTENT);
        params.addRule(RelativeLayout.ALIGN_PARENT_BOTTOM, RelativeLayout.TRUE);
        contentView.addView(mAdView, params);
    }

    public void onCreate() {
        try {
            if (mAdView != null) {
                mAdView.loadAd(new AdRequest.Builder().build());
            }
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

    public void onResume() {
        try {
            if (mAdView != null) {
                mAdView.resume();
            }
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

    public void onPause() {
        try {
            if (mAdView != null) {
                mAdView.pause();
            }
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

    public void onDestroy() {
        try {
            if (mAdView != null) {
                mAdView.destroy();
            }
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: app/src/LightningLite/java/acr/browser/lightning/ad/AdManager.kt
package acr.browser.lightning.ad

import acr.browser.lightning.R
import android.app.Activity
import android.view.ViewGroup
import android.widget.RelativeLayout
import com.google.android.gms.ads.*

class AdManager(activity : Activity) {
    private val adView : AdView? = AdView(activity)

    init {
        MobileAds.initialize(activity, "ca-app-pub-7879734750226076~3453529948")

        val contentView = activity.findViewById<ViewGroup>(R.id.ui_layout)

        adView?.adUnitId = "ca-app-pub-7879734750226076/6406996343"
        adView?.adSize = AdSize.BANNER
        adView?.adListener = AdListener()

        val params = RelativeLayout.LayoutParams(RelativeLayout.LayoutParams.MATCH_PARENT, RelativeLayout.LayoutParams.WRAP_CONTENT)
        params.addRule(RelativeLayout.ALIGN_PARENT_BOTTOM, RelativeLayout.TRUE)

        contentView.addView(adView)
    }

    fun onCreate() {
        adView?.loadAd(AdRequest.Builder().build())
    }

    fun onResume() {
        adView?.resume()
    }

    fun onPause() {
        adView?.pause()
    }

    fun onDestroy() {
        adView?.destroy()
    }
}